packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1,82 +1,60 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
 --- packages/rancher-monitoring/charts-original/Chart.yaml
 +++ packages/rancher-monitoring/charts/Chart.yaml
-@@ -1,22 +1,28 @@
+@@ -1,5 +1,11 @@
  apiVersion: v1
- appVersion: 0.38.1
--description: Provides easy monitoring definitions for Kubernetes services, and deployment
--  and management of Prometheus instances.
+-description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
 +annotations:
 +  catalog.cattle.io/certified: rancher
 +  catalog.cattle.io/namespace: cattle-monitoring-system
 +  catalog.cattle.io/release-name: rancher-monitoring
 +  catalog.cattle.io/ui-component: monitoring
 +  catalog.cattle.io/provides-gvr: monitoring.coreos.com.prometheus/v1
-+description: Modifies the upstream Prometheus Operator chart, which provides easy monitoring definitions for Kubernetes services and the deployment and management of Prometheus instances, and enables Prometheus Adapter on a default Prometheus instance.
- engine: gotpl
- home: https://github.com/coreos/prometheus-operator
++description: Modifies the upstream Prometheus Operator chart, which collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator, and enables Prometheus Adapter on a default Prometheus instance.
```

Comment:
nit: not sure what this part means and if we need it?
> which collects Kubernetes manifests

Reply:
This part comes straight from the Chart description on the new upstream in https://github.com/prometheus-community/helm-charts/blob/97e66d826c887b8cb1007d58c2e8cc7f0f7d645a/charts/kube-prometheus-stack/Chart.yaml#L2.

I think the intention of this is to say that Prometheus Operator also packages certain independent charts / manifests (i.e. `kube-state-metrics`, `node-exporter`, and `grafana` for upstream, `rancher-pushprox` and `prometheus-adapter` in addition for us) to deploy a full monitoring stack onto the cluster. These are declared as subcharts, but aren't technically required to deploy Prometheus Operator itself, so they are like independent manifests deployed alongside the chart.